comp/core/flare/flare.go

```diff
@@ -51,6 +52,7 @@ type dependencies struct {
 	WMeta                 optional.Option[workloadmeta.Component]
 	Secrets               secrets.Component
 	AC                    optional.Option[autodiscovery.Component]
+	RCSetting             rcsetting.Component
```

Comment:
Why do we need `rcsetting` if we don't use them anywhere ?

Reply:
My understanding is that newflare from remoteconfig ultimately pass rcsetting down to makeflare here and we need rcsetting to run :
https://github.com/DataDog/datadog-agent/blob/remote-config-stream-log/cmd/agent/subcommands/flare/command.go#L360

Reply:
`makeFlare` is in the CLI, here we're modifying the flare comp itself. I don't think remote-config uses the CLI when triggering flares. Also, if we're not using it in the component we don't need to request it.